services/big_panda.rb

```diff
@@ -0,0 +1,65 @@
+# See https://www.bigpanda.io/docs/integrations/index.html#alerts-rest-api
+class Service::BigPanda < Service
+  def receive_validate(errors = {})
+    success = true
+    [:app_key, :application].each do |k|
+      if settings[k].to_s.empty?
+        errors[k] = 'Is required'
+        success = false
+      end
+    end
+    success
+  end
+
+  def receive_alert
+    raise_config_error unless receive_validate({})
+    send_alert(body.merge('status' => 'critical'))
+  end
+
+  def receive_alert_clear
+    raise_config_error unless receive_validate({})
+    send_alert(body.merge('status' => 'ok'))
+  end
+
+  def send_alert(body)
+    url = 'https://api.bigpanda.io/data/v2/alerts'
+    begin
+      http_post(url, body, headers)
+    rescue Faraday::Error::ConnectionFailed
+      log("Connection failed for url: #{url} for payload: #{payload.inspect}")
+    end
+  end
+
+  def body
+    body = {
+      'app_key' => settings[:app_key],
+      'application' => settings[:application],
```

Comment:
P.S. It would be nice to include `'payload' => payload`. If that's too noisy we could remove it before merging, but I'm curious to see what BigPanda does with it.